vars/mavenCanaryRelease.groovy

```diff
@@ -66,3 +70,10 @@ def call(body) {
 
     contentRepository(body);
   }
+
```

Comment:
Please add a comment explaining what this method tries to achieve
e.g. 

Given a pom.xml as 
```xml
<foo>
</foo>

it returns 
<foo>
  <bar>
  </bar>
<foo>
```

Reply:
fixed it